x-pack/plugins/lens/public/editor_frame_plugin/editor_frame/workspace_panel_wrapper.tsx

```diff
@@ -0,0 +1,37 @@
+/*
```

Comment:
This file is in the wrong folder, should be `x-pack/legacy/plugins/lens/public/editor_frame_plugin/editor_frame/workspace_panel_wrapper.tsx`